internal/testrunner/runners/pipeline/test_config.go

```diff
@@ -20,6 +20,10 @@ type testConfig struct {
 	Multiline     *multiline             `json:"multiline"`
 	Fields        map[string]interface{} `json:"fields"`
 	DynamicFields map[string]string      `json:"dynamic_fields"`
+
+	// NumericKeywordFields holds a list of fields that have keyword
+	// type but can be ingested as numeric type.
+	NumericKeywordFields []string `json:"numeric_keyword_fields"`
```

Comment:
This way you're violating the package spec by introducing an additional field. Please go first with the PR to the package-spec. Unfortunately you can't simply define a field with an extension to spec.

EDIT:

Just another thought: I might not be on the same page, but could you explain why do you need this special property? I thought that this is a general rule that can be applied to all numeric fields. /cc @ycombinator